### PR TITLE
refactor($interpolate): remove unnecessary else

### PR DESCRIPTION
**Pull request by Nikhil22**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Cleaner code.

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?**

No.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

**Comment by Nikhil-Bhaskar** [comment]
I signed it

File: src/ng/interpolate.js [hunk 1]
```diff
@@ -91,9 +90,8 @@ function $InterpolateProvider() {
     if (value) {
       endSymbol = value;
       return this;
-    } else {
-      return endSymbol;
-    }
+    }
```
Comment (Narretz) [inline src/ng/interpolate.js]: There must be a trailing space here. Can you please remove it?

**Comment by Narretz** [comment]
Can you please update the commit message to `refactor(\$interpolate): remove unnecessary else` ?

File: src/ng/interpolate.js (hunk 1 above)
Comment (Nikhil22) [inline src/ng/interpolate.js]: it has been removed now

**Pull request merged**